build.gradle

```diff
@@ -48,6 +48,9 @@ subprojects {
             // https://sonarcloud.io/project/issues?id=Together-Java_TJ-Bot&issues=AXvleoajNC7zjjhmSufQ&open=AXvleoajNC7zjjhmSufQ
             // https://sonarcloud.io/project/issues?id=Together-Java_TJ-Bot&issues=AXvleoX5NC7zjjhmSufP&open=AXvleoX5NC7zjjhmSufP
             message 'java:S5998'
+
+            // Disables "Reflection should not be used to increase accessibility of classes, methods, or fields" rule (Reflection is required for the Javap class to work).
+            message 'java:S3011'
```

Comment:
Dont suppress this globally. Move this suppression over to the `Javap` class and suppress it there on a method level only, keep the scope of this as small as possible.

Reply:
fixed